src/pages/challenges.tsx

```diff
@@ -0,0 +1,80 @@
+import { Component } from 'react';
+import Layout from '../components/Layout';
+import Section from '../components/section';
+import Modal from '../components/modal';
+import Navbar from '../components/navbar';
+import Title from '../components/title';
+
+class Challenges extends Component<any> {
+  state = {
+    showModal: false,
+  };
+
+  btnClickHandler = () => {
+    const showModalVal = this.state.showModal;
+    this.setState({ showModal: !showModalVal });
+  };
+
+  closeModal = () => {
+    this.setState({ showModal: false });
+  };
+
+  render() {
+    const CHALLENGES = [
+      {
+        id: 1,
+        title: 'Sherlock and Anagrams',
+        level: 'Easy',
+        startDate: '14 Days Ago',
+        endDate: 'Today',
+        participantsCnt: '5',
+        isActive: 1,
+      },
+      {
+        id: 2,
+        title: 'Sherlock and Anagrams',
+        level: 'Easy',
+        startDate: '28 Days Ago',
+        endDate: '14 Days Ago',
+        participantsCnt: '5',
+        isActive: 0,
+      },
+    ];
+
```

Comment:
Let's move mock data into `mocks/challenges.json`

Reply:
Sure, I will move this accordingly